### PR TITLE
Changed JAVA_HOME to target included JDK.

### PR DESCRIPTION
**Pull request by dcap0**
Causing an error: 
```
ERROR: JAVA_HOME is set to an invalid directory: /snap/gradle/120/usr/lib/jvm/default-java

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.
````

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ apps:
       LANG: C.UTF-8
       LANGUAGE: C.UTF-8
       LC_ALL: C.UTF-8
-      JAVA_HOME: $SNAP/usr/lib/jvm/default-java
+      JAVA_HOME: $SNAP/usr/lib/jvm/java-1.16.0-openjdk-amd64
```
Comment (tunix) [inline snap/snapcraft.yaml:28]: Please use $SNAP_ARCH instead of `amd64`

**Review by tunix — CHANGES_REQUESTED** [review]
Thanks for your contribution! It's best to not exclude other possible archs.

File: snap/snapcraft.yaml (hunk 1 above)
Comment (tunix) [reply snap/snapcraft.yaml:28]: @dcap0 - how about we add a symlink?

```
$SNAP/usr/lib/jvm/default-java -> $SNAP/usr/lib/jvm/java-1.16.0-openjdk-$SNAP_ARCH
```

Comment (galgalesh) [reply snap/snapcraft.yaml:28]: imo, changing the env variable is preferred over creating a symlink because it removes a level of indirection.

**Comment by tunix** [comment]
I'll merge it as it's. Thanks!

**Review by tunix — APPROVED** [review]

**Pull request merged**